apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -678,6 +678,85 @@ static Extension createExtensionCoding(
     return extension;
   }
 
+  /**
+   * Adds a care team extension to the supplied careTeamComponent if there is not already an
+   * extension for the supplied extensionValue and extensionValue is not empty.
+   *
+   * @param codebookVariable the codebook variable to make the reference url
+   * @param extensionValue the value for the extension, typically sourced from the claimLine
+   * @param careTeamComponent the care team component to look for the extension in
+   * @param eob the eob
+   */
+  public static void addCareTeamExtension(
+      CcwCodebookVariable codebookVariable,
+      Optional extensionValue,
```

Comment:
Why is the `Optional` untyped?

Reply:
Since the optionals get all turned into strings, figured it'd be easier to just take the raw optional than have optionals cast to a bunch of types, so the method could be reused for any field type. I could see the argument for stricter typing with multiple overloads though (If I was reviewing this code Id probably push for that tbh)

Reply:
Looking again all the optionals that call this are Optional<String> so I'll just fix this.

Reply:
Fixed them to be `Optiona<?>` at least, to attempt to keep the flexibility.